webpage/grammar_checker.py

```diff
@@ -38,11 +38,11 @@ def correction(self,word):
     # generate possible spelling correciton for word
     def candidates(self,word):
         if len(word)<5:
-            return self.known([word]) or self.known(self.edits1(word)) or self.known(self.edits2(word)) or self.known(self.edits3(word)) or set([word])
```

Comment:
For word with length less than 5 only returned itself, which is incorrect.

Reply:
how many words should we do it for

Reply:
1️⃣ + number of words with edit 1️⃣.

Reply:
fixed with 99ad287bea7a215594c26c057b817c948f0cbb5e